Engine/TestStep.cs

```diff
@@ -1064,6 +1075,7 @@ void completeAction(Task runTask)
                         }
 
                         planRun.AddTestStepRunCompleted(stepRun);
+                        stepRun.SignalCompleted();
```

Comment:
Will the Completed event be set if this throws?

Reply:
It is not likely that an exception will be thrown.